src/java.base/share/classes/java/security/doc-files/security-related-system-properties.html

```diff
@@ -0,0 +1,192 @@
+ by Oracle in the LICENSE file that accompanied this code.
+
+ This code is distributed in the hope that it will be useful, but WITHOUT
+ ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ version 2 for more details (a copy is included in the LICENSE file that
+ accompanied this code).
+
+ You should have received a copy of the GNU General Public License version
+ 2 along with this work; if not, write to the Free Software Foundation,
+ Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+
+ Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ or visit www.oracle.com if you need additional information or have any
+ questions.
+-->
+<html lang="en">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>The java.security.debug system property</title>
+    <style>
+        #debug {
+         border: 1px solid black;
+         border-collapse: collapse;
+         margin: 0 auto;
+        }
+        #debug caption {
+          font-weight: bold;
+          font-size: smaller;
+        }
+        #debug, #debug th, #debug td {
+         border: 1px solid black;
+         padding: 2px .5em;
+        }
+        #debug tbody th {
+         font-weight: normal;
+         text-align:left;
+        }
+    </style>
+</head>
+<body LANG="en-US" DIR="LTR">
+<H1 style="text-align:center">The java.security.debug system property</H1>
+<a id="security.debug"></a>
+<H2>Debug</H2>
+<p><B>{@systemProperty java.security.debug}</B><BR></p>
+<p> To monitor security access, you can set the <code>java.security.debug</code> system property,
+    which determines what trace messages are printed during execution. The value of the property is one or more options separated by a comma.
+    Some options also have additional sub-options. Sub-options are specified by appending a ":" to the option, followed by a list of one more sub-options separated by a comma.
+    For example, to specify the <code>ocsp</code> and <code>verbose</code> options with the <code>certpath</code> option: set the property to "<code>certpath:ocsp,verbose</code>".
+</p>
+
+<H3>Printing Thread and Timestamp Information</H3>
+<p>
+    You can append the following strings to the value specified in the <code>java.security.debug</code>
+    system property to print additional information:
+<ul>
+    <li><code>+thread</code>:Print thread and caller information</li>
+    <li><code>+timestamp</code>:Print timestamp information</li>
+</ul>
+<p>
+    For example, to add thread, caller, and timestamp information to all debuging output,
+    set the <code>java.security.debug</code> system property on the command line as follows:
+    <pre><code>java -Djava.security.debug=all+thread+timestamp MyApp</code></pre>
+
+<p>The following table lists <code>java.security.debug</code> options table:</p>
```

Comment:
s/lists/lists the/

Remove "table" at the end, you already have that word at the beginning.